playwright/e2e/clusters/cluster-archives-list.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from '../../fixtures/pages';
+
+test.describe.serial('OCM Cluster archives page', { tag: ['@ci', '@smoke'] }, () => {
```

Comment:
why do we always use `serial`?

i'd expect it to only be needed in cases where tests are dependent on one another (which we shouldn't have anyway)